test/matching/runtests.jl

```diff
@@ -56,6 +56,57 @@ match = Matching.maximum_weight_maximal_matching(g,w,0)
 @test match.mate[5] == 2
 @test match.mate[6] == 1
 
+g = CompleteGraph(3)
+w =Dict{Edge,Float64}()
+w[Edge(1,2)] = 1
+w[Edge(3,2)] = 1
+w[Edge(1,3)] = 1
+match = Matching.maximum_weight_matching(g,w)
+@test match.weight == 1
+
+g = Graph(4)
+w =Dict{Edge,Float64}()
+add_edge!(g, 1,3)
+add_edge!(g, 1,4)
+add_edge!(g, 2,4)
+w[Edge(1,3)] = 1
+w[Edge(1,4)] = 3
+w[Edge(2,4)] = 1
+match = Matching.maximum_weight_matching(g,w)
```

Comment:
can you remove Matching. from here and from the other lines in this file (also a check to see that everything is correctly exported)